pkg/utils/filters/filter_test.go

```diff
@@ -50,3 +50,51 @@ func TestParseFilter(t *testing.T) {
 		}
 	}
 }
+
+func TestValidate(t *testing.T) {
+	type args struct {
+		filter map[string][]string
+	}
+	tests := []struct {
+		name    string
+		args    args
+		wantErr bool
+	}{
+		{
+			name: "",
```

Comment:
How about make `name` readable?

Reply:
cannot agree more. Thanks. and updated. @Ace-Tang